src/System.DirectoryServices/src/System/DirectoryServices/SearchWaitHandler.cs

```diff
@@ -15,34 +15,34 @@ namespace System.DirectoryServices
     using System.Reflection;
     using System.Security.Permissions;
     using System.Globalization;
+    // https://support.microsoft.com/en-us/kb/833789
```

Comment:
Is this code that will be permanently excluded or is it code that should at some point be fixed and uncommented?

If the former is true, then it should be deleted.